modules/build-run/pages/build-application.adoc

```diff
@@ -172,7 +172,9 @@ It is possible to override an existing file using the `-Dparameters.overwrite` s
 
 === Merge parameters configuration
 
-During the build of the application the `merge-configuration` goal is executed and the content of the `parameters-${bonita.environment}.yml` file is merged into the final Bonita configuration file.
+During the build of the application the `merge-configuration` goal (https://bonitasoft.github.io/bonita-project-maven-plugin/merge-configuration-mojo.html[bonita-project:merge-configuration]) is executed automatically.
+
+The content of the `parameters-${bonita.environment}.yml` file located in `.bcd_configurations` folder of your project is merged into the final Bonita configuration file.
```

Comment:
This is weird to talk about BCD here whereas we did not talk about it before.
At least it should be introduced.

Reply:
Moreover, it should be stated that this part is for Subscription only

Reply:
> Moreover, it should be stated that this part is for Subscription only

It's not... bonita-project-maven-plugin is public.
https://github.com/bonitasoft/bonita-project-maven-plugin/blob/c05767b60e00c0768f435915ec300a942491230c/plugin/src/main/java/org/bonitasoft/plugin/build/AbstractConfigurationArchiveMojo.java#L45
`.bcd_configurations` is just a folder name, which was chosen for compatibility to ease the migration to users of the old BCD product. But BCD is no longer a product, only an approach.

Reply:
this also impies we can't anymore use the extract conf to a display content of a .bconf file in a readable yaml file

Reply:
@rbioteau Insights about all this ?

Reply:
> this also impies we can't anymore use the extract conf to a display content of a .bconf file in a readable yaml file

It is not the purpose of the `bconf` file. You may still extract them to retrieve its content as they are just zip file but this is internal implementation and subject to changes without notice.